core/src/androidMain/kotlin/Bluetooth.kt

```diff
@@ -41,7 +46,21 @@ private val Context.locationManager: LocationManager
 private val isLocationEnabled: Boolean
     get() = LocationManagerCompat.isLocationEnabled(applicationContext.locationManager)
 
-internal actual val bluetoothAvailability: Flow<Bluetooth.Availability> =
+private val locationEnabledFlow = when {
+    SDK_INT > R -> flowOf(true)
+    else -> broadcastReceiverFlow(IntentFilter(PROVIDERS_CHANGED_ACTION))
+        .map { intent ->
+            if (SDK_INT == R) {
+                intent.getBooleanExtra(EXTRA_PROVIDER_ENABLED, false)
+            } else {
+                isLocationEnabled
+            }
+        }
+        .onStart { emit(isLocationEnabled) }
```

Comment:
@twyatt I was wondering if 
.onStart { emit(isLocationEnabled) }

should be changed to 

.onStart { emit(isLocationEnabled || SDK_INT > R) }

I think I tested it and found it should not be necessary.

Reply:
It isn't necessary because the `locationEnabledFlow` `when` block will use the `flowOf(true)` when the SDK > `R`.
In other words, the `broadcastReceiverFlow` isn't utilized at all on SDK > `R`.

Reply:
@twyatt Thank you for the information.